### PR TITLE
5220: Editable 'Legal notice' page

### PR DESCRIPTION
**Pull request by claudiu-cristea**

### CODE REVIEW

File: web/profiles/joinup/joinup.behat.inc [hunk 1]
```diff
@@ -2659,4 +2667,39 @@ JS;
...
+      'body' => [
+        'value' => 'Legal notice text.',
+        'format' => 'content_editor',
+      ],
+    ]);
+    $block->save();
+    $this->contentBlocks[$block->id()] = $block;
+    // As the page that contains the block is created before the block, we need
+    // a cache clear to account the presence of the new block.
+    drupal_flush_all_caches();
```
Comment (pfrenssen) [inline web/profiles/joinup/joinup.behat.inc]: Wow isn't this overkill? Can't we just clear the page cache of this page using the cache tag?

Having the whole cache unnecessarily cleared in the middle of a Behat tests slows things down by several seconds.

File: web/profiles/joinup/joinup.post_update.php [hunk 2]
```diff
@@ -42,3 +47,58 @@ function joinup_post_update_install_joinup_rss() {
...
+<p>This disclaimer is not intended to limit the liability of the Commission in contravention of any requirements laid down in applicable national law nor to exclude its liability for matters which may not be excluded under that law.</p>
+<h2>Privacy Statement</h2>
+<h3>The Specific electronic Service: Joinup</h3>
+<p>The objective of this portal is to facilitate the development, sharing and re-use of interoperability solutions for public administrations as well as the sharing of best practices in domains relevant to the public sector.</p>
+BODY;
+
+  // Create the 'Legal notice' block.
+  BlockContent::create([
+    'type' => 'simple_block',
+    'uuid' => 'ec092d17-ef18-42b0-b460-642871150cd3',
```
Comment (pfrenssen) [inline web/profiles/joinup/joinup.post_update.php]: Can we put this UUID in a constant?

File: web/profiles/joinup/joinup.behat.inc (hunk 1 above)
Comment (claudiu-cristea) [reply web/profiles/joinup/joinup.behat.inc]: Fixed. Clearing now only a cache item.

File: web/profiles/joinup/joinup.post_update.php (hunk 2 above)
Comment (claudiu-cristea) [reply web/profiles/joinup/joinup.post_update.php]: Done.

**Pull request merged**